libs/wingsdk/src/std/string.ts

```diff
@@ -45,7 +45,7 @@ export class String {
    */
   public contains(searchString: string): boolean {
     searchString;
-    throw new Error("Macro");
+    throw new Error("Abstract");
```

Comment:
Shouldnt this still throw "Macro"? I think we are using the Error("Macro") anywhere where we use macro overrides.